homeassistant/components/mqtt/const.py

```diff
@@ -27,6 +27,7 @@
 CONF_WS_PATH = "ws_path"
 CONF_WS_HEADERS = "ws_headers"
 CONF_WILL_MESSAGE = "will_message"
+CONF_PAYLOAD_RESET = "payload_reset"
```

Comment:
I saw `number` use it, and decided to de-duplicate it by moving it to `const` and instead import it in both modules.